src/App.tsx

```diff
@@ -40,8 +40,8 @@ export default function App() {
         <Route path='/' element={<Home/>}/>
         <Route path='/workflows/' element={<Workflows/>}/>
         <Route path='/workflows/:workflow' element={<WorkflowHistory/>}/>
-        <Route path='/workflows/:flowId/:runNumber/:taskId/:tab' element={<Run lineageData={configData}/>}/>
-        <Route path='/workflows/:flowId/:runNumber/:taskId/:tab/:stepName' element={<Run lineageData={configData} panelOpen={true}/>}/>
+        <Route path='/workflows/:flowId/:runNumber/:taskId/:tab' element={<Run configData={configData}/>}/>
```

Comment:
Why does it need configData?
The Graph is created from input/outputIds from the state files, right?

Reply:
That's true, the lineage in the "run" page is constructed from in/outputIds. However we use "configData" in order to check if the action exists in the current config, so that we can jump from the "Workflow explorer" to the "Config Viewer" by clicking on an edge